src/ape/api/providers.py

```diff
@@ -1032,7 +1032,18 @@ def get_receipt(
         except TimeExhausted as err:
             raise TransactionNotFoundError(txn_hash) from err
 
-        txn = dict(self.web3.eth.get_transaction(HexStr(txn_hash)))
+        max_retries = 20
```

Comment:
Can this be a config item?

Reply:
We probably want this to be a provider-level config right (as opposed to network)? 

Is 20 alright as the default value?

Reply:
provider-level would require updating all providers, I think network-level makes more sense (e.g. update ape-ethereum) since it's most likely to be an issue for a faster chain